onnxruntime/test/onnx/runner.h

```diff
@@ -101,7 +104,7 @@ class SeqTestRunner : public DataRunner {
  public:
   SeqTestRunner(OrtSession* session1, const ITestCase& c, size_t repeat_count, TestCaseCallBack on_finished1);
 
-  void Start(ORT_CALLBACK_INSTANCE pci, size_t concurrent_runs) override;
+  void Start(ORT_CALLBACK_INSTANCE pci, size_t concurrent_runs) noexcept override;
```

Comment:
But it could fail. It if is no except, then it should either have a returned Status or abort() on failure.